vsintegration/src/FSharp.LanguageService/LanguageServiceUtils.fs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.LanguageService
+
+open System
+open System.Configuration
+open System.Diagnostics
+
+module internal FSharpCommonConstants =
+    [<Literal>]
+    let packageGuid = "871D2A70-12A2-4e42-9440-425DD92A4116"
+    [<Literal>]
+    let languageServiceGuid = "BC6DD5A5-D4D6-4dab-A00D-A51242DBAF1B"
+    [<Literal>]
+    let editorFactoryGuid = "4EB7CCB7-4336-4FFD-B12B-396E9FD079A9"
+    [<Literal>]
+    let codePageEditorFactoryGuid = "82A16493-EF43-47E0-B42D-D87BAAB5335D"
+    [<Literal>]
+    let svsSettingsPersistenceManagerGuid = "9B164E40-C3A2-4363-9BC5-EB4039DEF653"
```

Comment:
Code style in other projects is to leave it to type inference. Any reason to change it?